main.py

```diff
@@ -103,7 +104,16 @@ def session_form(session, submitOnLoad=False):
     result = session.get('form_format_d', {})
     trigger = "input delay:200ms, load" if submitOnLoad and result else "input delay:200ms"
     session_form = Form(hx_post='/submit', hx_target='#result', hx_trigger=trigger, id="session_form", hx_swap_oob="#session_form", style=form_style)(
-                    P("Please complete all fields below to describe the machine learning model metadata."),
+                    P(
+                        "The STAC Machine Learning Model (MLM) metadata specification makes it easy to describe the metadata needed to reproduce model inference "
+                        "and enable search and discovery. The MLM makes it easier to share, reuse, and run models "
+                        "on inference providers that support it, such as",
+                        A("Wherobots", href="https://wherobots.com/wherobotsai-for-raster-inference/", target="_blank", rel="noopener noreferrer", cls="border-b-2 border-b-black/30 hover:border-b-black/80"),
+                        ", and open source ml frameworks.\n\n"
+                        "Please complete all required fields below prior to copying or downloading the JSON result. For more information on the specification, refer to the ",
```

Comment:
A sentence on the downloaded json would be good. like downloaded JSONs will be stored in your download directory with the name of your model asset or something similar

Reply:
done!